manifests/server.pp

```diff
@@ -79,6 +79,7 @@
   Boolean $handle_creds                                 = $mongodb::params::handle_creds,
   Boolean $store_creds                                  = $mongodb::params::store_creds,
   Array $admin_roles                                    = $mongodb::params::admin_roles,
+  Optional[Boolean] $repl_enable_majority_read_concern  = undef,
```

Comment:
Could you make this non optional and default to true? That makes the code here and in the template easier to read.